<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingForm</class>
 <widget class="QWidget" name="SettingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,3">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>任务有效设定</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radioButton_MSB">
            <property name="text">
             <string>MSB存储</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_LSB">
            <property name="text">
             <string>LSB存储</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radioButton_CharT1">
            <property name="text">
             <string>阳文(字1空0)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_CharT0">
            <property name="text">
             <string>阴文(字0空1)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox_HorFirst">
            <property name="text">
             <string>水平镜像</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_VerFirst">
            <property name="text">
             <string>竖直镜像</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>测试</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>网格数量：</string>
            </property>
            <property name="buddy">
             <cstring>griddingNumber</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="griddingNumber"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>设定显示文字：</string>
            </property>
            <property name="buddy">
             <cstring>mainChar</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="mainChar"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFontComboBox" name="fontComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_ShowChar">
          <property name="text">
           <string>是否显示文字</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CharWidget" name="charWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>2</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CharWidget</class>
   <extends>QWidget</extends>
   <header>charwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>radioButton_MSB</tabstop>
  <tabstop>radioButton_LSB</tabstop>
  <tabstop>checkBox_HorFirst</tabstop>
  <tabstop>checkBox_VerFirst</tabstop>
  <tabstop>radioButton_CharT1</tabstop>
  <tabstop>radioButton_CharT0</tabstop>
  <tabstop>griddingNumber</tabstop>
  <tabstop>mainChar</tabstop>
  <tabstop>checkBox_ShowChar</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
